src/components/proofEditor/proofEditor.tsx

```diff
@@ -3,7 +3,7 @@ import { checkProof } from "../../util/proofChecker";
 import styles from "./proofEditor.module.scss";
 
 export interface Props {

```

Comment:
```suggestion
// tslint:disable-next-line:no-empty-interface
export interface Props {
```

This will ignore the empty interface linter error.

Reply:
Another option could be to remove the Interface entirely and change line 13 to `... extends React<State> {`

I think the interface is there to make the code more readable and to predefine State and Props and I think it helps. So I would keep it, since it let's us now that there are no props passed.

Reply:
I agree that disabling the linterrule for this would be one solution. I will apply it for now.